xscen/catutils.py

```diff
@@ -104,13 +106,26 @@ def _parse_datebounds(text: str) -> tuple[str, str]:
 
 
 def _find_assets(
-    root: Union[str, Path],
+    root: Union[str, os.PathLike],
     exts: set[str],
     lengths: set[int],
-    dirglob: Optional[str] = None,
+    dirglob: str = None,
```

Comment:
This change means that passing `dirglob = None` explicitly is no longer allowed.